modules/special-rules/pages/damaging-flyers.adoc

```diff
@@ -0,0 +1,19 @@
+= Damaging flyers
+
+Successful rolls against airborne flyers work differently to other units -- we use an extra step to determine what actually happens.
```

Comment:
> I think that this is problematic. First, "Successful rolls..." of what? It's actually about successful rolls to hit, isn't it? Hence "Hits on airborne flyers..." in the previous draft under `hits-on-flyers.adoc`.

NO!

The entire point of this rewrite is that _you haven't scored a hit on the Flyer yet_. Which is why the Snap Fire sections talk about discarding any dice rolls lower than the lowest Armour value in the Flyer detachment and then allocating dice. With the structure borrowed from the rewritten hit allocation section.

The aim here is to remove the confusion that you had during your last or last-but-one game, about what a 'hit' on a Flyer meant.

Reply:
The act of 'rolling to hit' is pretty well embedded. But a 'good' roll to hit doesn't *necessarily* lead to the target 'taking a hit' and thus 'losing a point of damage capacity and/or being destroyed' — and so it can get a bit tricky to phrase.

The way I see it, you've taken the opposite approach to me in this case:

- I've tried to clarify that flyers are special in that a 'hit' doesn't automatically mean 'cause a point of damage' as it does in most other cases (barring a save). Rather, there's another step to resolve *after* the 'hit' — procedurally, somewhat in kind with how we deal with saves and damage capacity *after* allocation of a hit.
- You've tried to reformulate successful attacks on flyers as some kind of 'potential hit' or 'damage that doesn't necessarily lead to taking a hit that would then mean destruction or loss of a point of damage capacity, barring a save'. In effect, you consider the second roll as the roll to hit and you're placing emphasis such that the extra step is *before* what you frame as the roll to hit. And then you're trying to reframe or tweak everything else to fit this.
 
I aim to keep an open mind, but so far I find your formulation much more complicated and confusing — and not just because it breaks with GW's original text.

To reframe attacks against aircraft, you've removed most mentions of 'hit' from the Flak rules (you missed the example with the Fire Prism, and had to make a special exception for drop pods). But, there are at least four problems with this I think:

1. It seems unintuitive — I believe that players will naturally still think of the flak Snap Fire attack as a roll to hit, even if you don't call it this and even if they weren't exposed to the original text which does call it a hit. These 'ifs' seem quite significant to me.
2. Flak rules still largely rest on Snap Fire rules — and **those** rules still refer to hits. That's a problem isn't it?
3. Your note makes special mention that we can use Flak against drop pods. That's fine, and probably necessary in some form. But given your new formulation, you're effectively forced to state that in this case, as they are not aircraft, the attack works like regular Snap Fire, with a single roll to hit and no extra step. It's a special exception to the 'not a roll to hit' attack roll... which is itself an exception to regular Snap Fire rule in the first place, where it is a roll to hit. You're creating an exception to an exception, just to revert back to the baseline.
4. Your new use of the term 'damage' is confusing. Elsewhere in the rules context, 'damage' strongly correlates with the meaning that the target takes a hit, and (barring any save that may apply) the unit is destroyed or loses a damage point as appropriate. I think it is unhelpful to mess with this correlation and introduce ambiguity through a new more general meaning of damage... especially when later in the same process the more specific and usual meaning does come into play.

This all seems very convoluted and unnecessary to me. It seems far more straightforward and intuitive to just clarify that hits on aircraft are slightly special in that we have an extra step to determine exactly what the effects are.

Reply:
> I aim to keep an open mind, but so far I find your formulation much more complicated and confusing — and not just because it breaks with GW's original text.

Like I said at https://github.com/GameBrains/er-core/pull/33#discussion_r841083775, I'm not wedded to 'damage' term, and I think you have given good arguments against using it.

But my starting point for the rewrite is that GW tried the 'hits are hits are hits, except against Flyers' approach, and it failed, because it appears to be widely misunderstood. Followed by you using the same approach in the first Remastered text, with almost exactly the same results. Unless I've got my times mixed up, the first Remastered text even caused confusion when you were reading your own text during a game.

As far as I'm concerned, two failed attempts at an approach is enough, and is a really big sign that we need to do approach it differently.

1. OK, you think it's unintuitive. The original GW approach is also unintuitive, and massively confusing because it takes a defined and consistently-used game term and uses it in a completely different way.
2. This is all covered in the rewritten mission texts, which specify exactly how Snap Fire interacts with Flyers. Whether that's Snap Fire from regular ground units, units with Flak, or units with Flak on Flak orders.
3. That note will be required anyway.
4. As above, I've already said that I have no issues with finding a better term.

> This all seems very convoluted and unnecessary to me. It seems far more straightforward and intuitive to just clarify that hits on aircraft are slightly special in that we have an extra step to determine exactly what the effects are.

We'll have to agree to disagree on which version is convoluted and unnecessary. An approach with two failed attempts so far means, for me, that a n alternative approach **is** necessary.

Reply:
>Unless I've got my times mixed up, the first Remastered text even caused confusion when you were reading your own text during a game.

As I said near the time, I couldn't remember the exact pickle we got into but our confusion had more to do with combination with another rule (it might've been something in the interception and/or flak procedures). At the outset, we were actually pretty clear on the 'hit on flyers' rules — but something else that didn't seem to reconcile made us start to question the whole thing. It's inadequate writing that undermines the concept — the concept itself is okay. So, I find your conclusion pre-emptive:

> As far as I'm concerned, two failed attempts at an approach is enough, and is a really big sign that we need to do approach it differently.

---

Rather than labouring over counter-arguments or clarifications on your other points — and potentially talking at cross purposes to each other again — I think this might be one of those instances where it'd be better to talk it through. I could make some time tomorrow or else towards the end of the week if it suits you?

Reply:
I can do half an hour around 3-4 this afternoon, otherwise it'll have to be towards the end of the week.